index.html

```diff
@@ -56,9 +56,11 @@
         </div>
     </header>
     <!-- hero -->
-    <section class="container hero">
-        <div class="row">
-            <div class="col-xs-12">
+    <section>
```

Comment:
Review one more time hero section HTML - spaces are not right and it looks like one </div> is missing

Reply:
perdariau didziaja dali logikos, dabar atrodo lyg nieko

Reply:
beje, tie tarpai buna kai rasai dvi klases prie vieno vivo, ju nereik perskirinet jokiais zenkliais.